lib/gooddata/models/domain.rb

```diff
@@ -348,8 +346,17 @@ def create_users(list, options = {})
       GoodData::Domain.create_users(list, name, { client: client }.merge(options))
     end
 
-    def segments(id = :all)
-      GoodData::Segment[id, domain: self]
+    def data_products(id = :all)
+      GoodData::DataProduct[id, domain: self]
+    end
+
+    def create_data_product(data)
+      data_product = GoodData::DataProduct.create(data, domain: self, client: client)
+      data_product.save
+    end
+
+    def segments(id = :all, data_product = nil)
```

Comment:
Doesn't depend on instance state (maybe move it to another class?) (UtilityFunction)